frame/staking-rewards/README.md

```diff
@@ -1,116 +1,187 @@
+
+A transfer of rewards obeys the same split.
+
+## Positions
+
+A position is what a user (or other protocol) gets when stakes amount. So that one can get a `share` of rewards and other benefits.
+
+### Staking
+
+Assuming a user has a configuration for `PICA` that allows us to stake for a month, she can `stake` an amount `X`` for once month duration with the amount larger than ED.
+The reward multiplier elevates the computed share of the staking pool, the longer you stake, the higher your share.
+Let's say we have a reward multiplier of`0.8`for a ``MONTH`, if I stake `X PICA`, my share will become `X * 0.8`.
+
+Once she staked `X` tokens, the pallet will create a `position` for an account.
+This position allows computing the user's share and the
+reward, that the user will be able to claim at any time.
+
+Penalties and lock periods are optional depending on pool configuration.
+
+### Staked asset
+
+User share amount, potentially elevated, issued as a new token onto asset account owned by staking position account.
+
+Each staking pool has its asset.
+
+All amounts s locked and unusable until wrapped into [fNFT](../nft/README.md).
+
+Later it may be less strict regarding locking derived amounts.
+
+### Unstaking while locked
+
+User may leave position before maturity (if it was defined as time lock), so he would likely pay penalty.
+
+This case is called `early unstake` and will result in a penalty applied to the
+staked asset. Calling the `unstake` extrinsic on a position that is still locked will
+unlock it in exchange for a fraction of the stake (configured for the staked
+asset).
+
+Assuming the penalty is defined as being `0.5`, if I staked `10000
+PICA`, unstaking will result in a penalty of  `5000 PICA`. All
+harvested rewards are still in possession of the user.
+
+the early unstake penalty, applied on the staked asset when the user unstake
+  early (before the end of the selected staking duration)
+
+It means that if a user will unstake before a lock ends, he will get half of locked amount. The other half will go to treasury
 
-Can be made only during epoch, but not during epoch transitions (for scalability and corretness reasons).
 
-#### Extend positions
+### Claiming
+
+A user can claim his pending rewards at any point in time.
 
-User can extend position with amount or with time.
+### Split position
+Allows splitting positons without paying a penalty.
 
-Can extending is not acted on current epoch, but merged during transitions.
+If user had position with MONTH lock total, with 2 weeks already passed with 100 tokens.
 
-##### Amount
+It can split position into several parts, examples 20, 30, 40 tokens. Each of which will be same lock duraion and time lock passed.
 
-Adds amount to stake. Diminishes time it would take until penalty by value calculated as follows:
+Splitting will not loose rewards for ongoing epoch.
+
+Can be made only during epoch, but not during epoch transitions (for scalability and corretness reasons).
```

Comment:
This is not(no longer) correct, splitting can happen anytime, please read the design doc at https://github.com/ComposableFi/composable/blob/b0d739202183fabf1c653d100da7f215d6dfdfed/rfcs/0004-pablo-distribution.md#535-split-position